package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "prestart": "npm run build",
-    "start": "node ./dist/server.js",
-    "dev": "DEBUG=commonwealth:* nodemon ./src/server.js --exec babel-node --watch src",
+
+    "start": "cross-env DEBUG=commonwealth:* node ./dist/server.js",
+    "dev": "cross-env DEBUG=commonwealth:* nodemon --debug ./src/server.js",
+    "dev-win": "cross-env DEBUG=commonwealth:* nodemon --debug ./src/server.js --exec babel-node",
     "build": "babel src --out-dir dist",
+    "build-win": "./node_modules/.bin/babel src --out-dir dist --source-maps",
```

Comment:
Num poderia só colocar `cross-env` aqui tambem nāo?

Reply:
A gente precisa do `--source-maps`?
Isso faz mais sentido para um aplicacaçāo front-end, para uma API seria relevante tbm?

Reply:
- Mesmo caso de cima. 

`--source-maps` foi no copy and paste, vou remover.